test/integration/suites/k8s-crd-mode/README.md

```diff
@@ -0,0 +1,11 @@
+# Kubernetes Suite
```

Comment:
Looks like this needs to be updated with the tests specifically done in this suite?